### ISSUE TITLE
Repo Cleanup

### ISSUE THREAD

**Issue by nsyousef** [cse110-sp23-group3/cse110-sp23-group3#84]
We should make sure our repo is cleaned up and prepared for submission. Some of the tasks we need to do are:

- [ ] delete any unused files from the repo
- [ ] have documentation explaining to Malcom (or whichever TA grades this) how to use our CI/CD build pipeline and how to generate release notes
- [ ] perhaps have documents explaining our branching strategy
- [ ] any other instructions we need to give Malcom so he can understand how we did things
- [ ] Update file tree structure in README.md in main (it does not auto-update and was generated a while ago (unless anyone updated it that I am not aware of)

If you all can think of anything else we need to do to clean up the repo, feel free to add it to this issue as well. I don't know if we will need to have this done by Sunday, or if we just need the coding done by Sunday and can do this during finals week. If someone knows, could you please clarify? Thanks!

**Comment by henryzhang03** [comment]
@nsyousef @phuanh004 just did some cleanup and updated the tree. Regarding the other documentation, I can write that later tonight.

**Comment by henryzhang03** [comment]
@nsyousef are you able to help write the CI/CD pipeline doc and the release notes one as well?

**Comment by nsyousef** [comment]
Sure. I can write those.

**Comment by nsyousef** [comment]
The CI/CD Pipeline and Release Notes Generator docs have been created. If you have time, please review them to make sure everything is clear. The CI/CD pipeline docs can be found in `admin/cipipeline` and the release notes generator docs can be found in `admin/releasenotesgenerator`.

**Comment by phuanh004** [comment]
Hmm, got some CI error with the latest commit

**Comment by nsyousef** [comment]
Yeah. I saw those too. I think one of the E2E tests is timing out, but not really sure why. Not sure if we should change it at this point because of the code freeze or if we should just let it be.

**Comment by henryzhang03** [comment]
It's not too big of a deal, change if you want. Its just Github's machines being slow lol. You just need to increase the timeout. @phuanh004 @nsyousef

**Comment by phuanh004** [comment]
It looks good to me.
